src/components/ATATContactForm.vue

```diff
@@ -108,6 +109,19 @@
           </v-col>
         </v-row>
 
+        <ATATTextField
+            v-if="showJobTitle"
+            id="JobTitle"
```

Comment:
This and all other inputs (and anything with an `id`) will need to have unique `id`s and now that there are 2 instances of this form on the page, you'll need to pass a prop to either prepend or append a string that will make them unique.